### PR TITLE
Feature/statsbar

### PR DESCRIPTION
**Pull request by GijsWeterings**
mooi status

### CODE REVIEW

File: app/elements/cardlist-api/card-chart.html [hunk 1]
```diff
@@ -7,8 +9,31 @@
         margin-top: 5px;
       }
 
-      svg-piechart {
+      /*svg-piechart {
```
Comment (TimvdLippe) [inline app/elements/cardlist-api/card-chart.html]: This can be removed

File: app/elements/cardlist-api/card-chart.html [hunk 2]
```diff
@@ -7,8 +9,31 @@
...
         margin: 15px;
+      }*/
+
+      material-progress-bars {
+        width: 90%;
+        margin: 20px 5%;
+        @apply(--paper-font-body2);
+      }
+      material-progress-bars > .bar > span {
```
Comment (TimvdLippe) [inline app/elements/cardlist-api/card-chart.html:17]: :see_no_evil: :see_no_evil: :see_no_evil: :see_no_evil: :see_no_evil:

Comment (GijsWeterings) [inline app/elements/cardlist-api/card-chart.html:17]: #elementsfromthecommunity

**Comment by TimvdLippe** [comment]
LGTM

**Pull request merged**